public/app/features/dashboard/dashgrid/PanelChrome.tsx

```diff
@@ -178,7 +178,7 @@ export class PanelChrome extends PureComponent<Props, State> {
         queries: panel.targets,
         panelId: panel.id,
         dashboardId: this.props.dashboard.id,
-        timezone: this.props.dashboard.timezone,
+        timezone: this.props.dashboard.getTimezone(),
```

Comment:
`this.props.dashboard.timezone` was always an empty string, could be a bug. But to get this working I called `this.props.dashboard.getTimezone()` instead.

Reply:
@torkelo @ryantxu any thoughts on this one?